### PR TITLE
APM-345714 version info in releases

### PR DESCRIPTION
**Pull request by karolkieglerski**

### CODE REVIEW

File: build/function-package.sh [hunk 1]
```diff
@@ -20,4 +20,4 @@ rm -f ./artefacts/dynatrace-gcp-function.zip
 (cd ./src/; zip -r ../artefacts/dynatrace-gcp-function.zip ./ -x '*__pycache__*')
 sed -i "s/^GCP_FUNCTION_RELEASE_VERSION=.*/GCP_FUNCTION_RELEASE_VERSION='$TRAVIS_TAG'/" scripts/setup.sh
 zip -r ./artefacts/function-deployment-package.zip ./ext_tools/
-zip -j ./artefacts/function-deployment-package.zip ./scripts/lib.sh ./scripts/setup.sh ./scripts/uninstall.sh ./activation-config.yaml
```
Comment (wojtasskorcz) [inline build/function-package.sh:23]: Can we change the typo in the path here from `artefacts` to `artifacts` or will it cause too much collateral damage?

Comment (mmajcher) [reply build/function-package.sh:23]: I like this expression "collateral damage". Never know what you will break 😆

Comment (karolkieglerski) [reply build/function-package.sh:23]: @wojtasskorcz where do you see typo? https://dictionary.cambridge.org/pl/dictionary/english/artefact

Comment (mmajcher) [reply build/function-package.sh:23]: "Artefact is the original British English spelling. Artifact is the American English spelling. "

Comment (karolkieglerski) [reply build/function-package.sh:23]: so everything is ok :)

Comment (wojtasskorcz) [reply build/function-package.sh:23]: Oh, didn't know. Never seen "artefact" in the wild :) All good then!

File: src/main.py [hunk 2]
```diff
@@ -103,6 +103,7 @@ async def handle_event(event: Dict, event_context, projects_ids: Optional[List[s
 
         dynatrace_api_key = await fetch_dynatrace_api_key(gcp_session=gcp_session, project_id=project_id_owner, token=token)
         dynatrace_url = await fetch_dynatrace_url(gcp_session=gcp_session, project_id=project_id_owner, token=token)
+        check_version(logging_context=context)
```
[the remark this answers is not in the document]
Comment (karolkieglerski) [reply src/main.py:106]: MetricsFastCheck  - docker for metrics 
LogsFastCheck - docker for logs
main.py - function

[the remark this answers is not in the document]
Comment (wojtasskorcz) [reply src/main.py:106]: What about `main.async_dynatrace_gcp_extension`? The docstring there says `Used in docker or for tests` and it calls `main.handle_event` within its body.
Admittedly, when I dug deeper into its usages, it's only used for metrics in Docker, so `LogsFastCheck` still makes sense. Invocation of `check_version` in `MetricsFastCheck` seems superfluous, though

[the remark this answers is not in the document]
Comment (karolkieglerski) [reply src/main.py:106]: done

**Review by wojtasskorcz — APPROVED** [review]

**Review by mswiatkowska — APPROVED** [review]

**Review by dtwwolinski — APPROVED** [review]

**Pull request merged**